forta-uniswap-event-py/src/agent_test.py

```diff
@@ -0,0 +1,192 @@
+        ],
+        "data": "0x0000000000000000000000000000000000000000000000005623309cafe37c00",
+        "log_index": 150,
+        "block_number": 13282824,
+        "block_hash": "0x85d8e4f37fd7146d82d3fdb851668ad88eb5351cd535c5fc3d62a40eed817c92",
+        "transaction_index": 136,
+        "transaction_hash": "0x0693912717239d8513f4f4af02d401011418381ec0a398d21d6b6dc0bd3d9486",
+        "removed": False,
+    }
+
+    withdrawal_event = {
+        "address": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+        "topics": [
+            "0x7fcf532c15f0a6db0bd6d0e038bea71d30d808c7d98cb3bf7268a95bf5081b65",
+            "0x0000000000000000000000007a250d5630b4cf539739df2c5dacb4c659f2488d",
+        ],
+        "data": "0x0000000000000000000000000000000000000000000000007375695a9e01ca7a",
+        "log_index": 239,
+        "block_number": 13283200,
+        "block_hash": "0xf07a5c6d6b1a412a321d21781736d77a39d15024336b9fe2b30dbc666308f550",
+        "transaction_index": 202,
+        "transaction_hash": "0x1e56a8fc69bae44e22e8de761a08a5d0740348fbe4a0d039262af4545e188614",
+        "removed": False,
+    }
+
+    swap_event = {
+        "address": "0xb4e16d0168e52d35cacd2c6185b44281ec28c9dc",
+        "topics": [
+            "0xd78ad95fa46c994b6551d0da85fc275fe613ce37657fb8d5e3d130840159d822",
+            "0x0000000000000000000000007a250d5630b4cf539739df2c5dacb4c659f2488d",
+            "0x0000000000000000000000007a250d5630b4cf539739df2c5dacb4c659f2488d",
+        ],
+        "data": "0x00000000000000000000000000000000000000000000000000000006088fce8e000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000007375695a9e01ca7a",
+        "log_index": 238,
+        "block_number": 13283200,
+        "block_hash": "0xf07a5c6d6b1a412a321d21781736d77a39d15024336b9fe2b30dbc666308f550",
+        "transaction_index": 202,
+        "transaction_hash": "0x1e56a8fc69bae44e22e8de761a08a5d0740348fbe4a0d039262af4545e188614",
+        "removed": False,
+    }
+
+    if event == "deposit":
+        return AttrDict(deposit_event)
+    elif event == "withdrawal":
+        return AttrDict(withdrawal_event)
+    elif event == "swap":
+        return AttrDict(swap_event)
+
+    return AttrDict({})
+
+
+def test_transaction_normal():
+    """
+    Send a benign transaction from one address to another
+    This should not raise an alert
+    """
+    # Send a normal transaction that doesnt emit a Deposit or Withdrawal event
+    tx_dict = gen_tx_data(to=ROUTER_ADDR)
+    tx_dict.update(gen_tx_receipt())
+
+    tx_event = create_transaction_event(tx_dict)
+    findings = handle_transaction(tx_event)
+
+    assert len(findings) == 0
+
+
+def test_transaction_deposit_event():
+    """
+    Send a transaction that trades a low amount of ether (.01 ether) for a token
+    This should not raise an alert as it is below the treshold
```

Comment:
This comment seems to disagree with the `assert` statement at the end.  It appears that the docstring needs to be updated.